### ISSUE TITLE
Style.Require strips query string

### ISSUE THREAD

**Issue by orchardbot** [OrchardCMS/Orchard#3954]
**woodvorg** created:
https://orchard.codeplex.com/workitem/20126

Running Orchard 1.6.
manifest.DefineStyle("Site").SetUrl("Site.css?d200913");

Style.Require("Site").AtHead();

The querystring "?d200913" is stripped at the output... but only if the file Site.css exists. If the file doesn't exist it doesn't strip the output.

**Comment by jbuck79** [comment]
This appears to have been fixed by #7564 but it hasn't been merged into master yet.

**Comment by sebastienros** [comment]
Fixed in dev

**Comment by madmox** [comment]
Tried migrating our solution to Orchard 1.10.3 to check if this issue was fixed. Looks like it's not!

This keeps the querystring:
```
Script.Include(Url.Content("foo.js?foo=bar")).AtHead()
// Generates: <script src="/Modules/MyModule/scripts/foo.js?foo=bar" type="text/javascript"></script>
```

When the file `foo.css` exists, this doesn't:
```
Style.Include(Url.Content("foo.css?foo=bar")).AtHead()
// Generates: <link href="/Modules/MyModule/Styles/foo.css" rel="stylesheet" type="text/css" />
```

**Comment by madmox** [comment]
Looks like the whole PR #7564 is not included in Orchard 1.10.3 (at least not in [the revision](https://github.com/OrchardCMS/Orchard/commit/a53d30dd6a94d80aa02ae98e617d288ffdc23f64) associated to the tag).
